### PR TITLE
Feat/autoposition

### LINKED ISSUE #932: position: Position/Tooltip miscalculation
> I think there's some kind of bug in the [Position|Tooltip] element (not sure which, but thinking Position). If you got to https://app-stage.pluralsight.com/plan-analytics/1800contacts/skills the info icon as shown below should show a tooltip. It works when the browser window has plenty of space to the right. When the browser window width shrinks, it takes a few mouse overs for the position to get calculated correctly. If you start out with a smaller browser width, the tip does not even show at all until you've moused in and out a lot while something is calibrating position and size and eventually the tip starts to show up, narrowing in on the correct position. (edited) 

cc https://pluralsight.slack.com/archives/C70HPSJPP/p1584550569101100

### PR DESCRIPTION
**Pull request by EdwardIrby**
resolves #932 #1036

This does not get rid of the tip. @sweeting should we do that in this PR https://pluralsight.slack.com/archives/C5CMEKH5Z/p1593465151076300

### CODE REVIEW

File: packages/position/src/react/__stories__/index.story.js [hunk 1]
```diff
@@ -232,3 +232,98 @@ function JsStory({ positionType, ...rest }) {
 JsStory.propTypes = {
   positionType: PropTypes.oneOf(Object.keys(positionFns)).isRequired
 }
+
+const edgeCaseSotries = storiesOf('Position Edge', module)
```
Comment (danethurber) [inline packages/position/src/react/__stories__/index.story.js:236]: typo 😉 `edgeCaseSotries `

File: packages/position/src/react/__stories__/index.story.js [hunk 2]
```diff
@@ -232,3 +232,98 @@ function JsStory({ positionType, ...rest }) {
...
+    const [b2hover, setb2hover] = React.useState(false)
+    const [b3hover, setb3hover] = React.useState(false)
+    const [b4hover, setb4hover] = React.useState(false)
+
+    return (
+      <div
+        style={{
+          display: 'grid',
+          justifyItems: 'start',
+          grid: '1fr 1fr/ 1fr 1fr',
```
Comment (danethurber) [inline packages/position/src/react/__stories__/index.story.js:251]: what does the `/` do in this style?

File: packages/position/src/js/__specs__/index.spec.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ describe('#below', () => {
     expect(below(targetEl).styleFor(el)).toEqual({
       position: 'absolute',
       top: '12px',
-      left: '-50px'
+      left: '0px'
```
Comment (danethurber) [inline packages/position/src/js/__specs__/index.spec.js:29]: i was hoping these snapshots wouldn't change. maybe we need to setup the window width/height for these as well

Comment (EdwardIrby) [reply packages/position/src/js/__specs__/index.spec.js:29]: Actually the point was for these to change. There should never be time when the tooltip is offscreen.

File: packages/position/src/react/__stories__/index.story.js (hunk 2 above)
Comment (EdwardIrby) [reply packages/position/src/react/__stories__/index.story.js:251]: It's required for the css prop grid. Grid is a shorthand rule and the / signifies the division between rows and columns when writing the rules.

**Pull request merged**